internal/network/utils.go

```diff
@@ -177,6 +177,34 @@ func GetClusterNetworkCidrs(cluster *common.Cluster) (ret []string) {
 	return
 }
 
+func GetApiVips(cluster *common.Cluster) (ret []string) {
+	ret = funk.Map(cluster.APIVips, func(x *models.APIVip) string { return string(x.IP) }).([]string)
+	return
+}
+
+func GetIngressVips(cluster *common.Cluster) (ret []string) {
+	ret = funk.Map(cluster.IngressVips, func(x *models.IngressVip) string { return string(x.IP) }).([]string)
+	return
+}
+
+func GetApiVipById(cluster *common.Cluster, index int) string {
```

Comment:
A proposal that we can do later if we agree that it would be a good idea: 
We agree that both `GetApiVipById` and `GetIngressVipById` should be failing silently by returning an empty string. However, it might be good to add a debug-level log for this.